packages/ui-components/scripts/generate-illustrations.js

```diff
@@ -59,11 +59,13 @@ async function main() {
 		const className = upperFirst(camelCase(key));
 		const parsedData = parse(defaultItem.data);
 
+		const htmlContent = toHTML(getRoot(parsedData)).replace(/style="mask-type:alpha"/g, 'style={{"mask-type": "alpha"}}');
```

Comment:
I think we should build a more generic way to handle future styles inside the SVG. So instead of regex this by only this style we should be open to other possible styles and build an object with all the content.